tests/Controller/__snapshots__/AssertionResponseControllerTest__test__if_it_fails_for_invalid_assertion_Response__1.yml

```diff
@@ -0,0 +1,14 @@
+-
+    - info
+    - 'Verifying if there is a pending authentication from SP'
+    - {  }
+-
+    - info
+    - 'Verify valid public key credential response'
+    -
+        nameId: JohnDoe123
```

Comment:
Have you further looked into whether or not we may log the nameID of the registering/authtenticating user?

Reply:
No, same as Tiqr. Added to my list to ask @joostd